pvlib/iotools/pvgis.py

```diff
@@ -136,6 +137,12 @@ def get_pvgis_hourly(latitude, longitude, start=None, end=None,
         the error message in the response will be raised as an exception,
         otherwise raise whatever ``HTTP/1.1`` error occurred
 
+    Info
+    ----
```

Comment:
```suggestion
    Notes
    -----
```
https://numpydoc.readthedocs.io/en/latest/format.html#notes